lib/usermanager.js

```diff
@@ -748,8 +761,28 @@ exports = module.exports = {
             return res.json(err);
           }
 
-          res.statusCode = 200;
-          return res.json(usr);
+          self.getUserRoles(usr, function(err, roles) {
+            if (err) {
+              res.statusCode = 500;
+              return res.json(err);
+            }
+
+            var data = usr.toObject();
+
+            if (roles.length > 0) {
+              data.rolesAsName = [];
+
+              for (var i = 0; i < roles.length; i++) {
+                data.rolesAsName.push(roles[i].name);
+              }
+            } else {
+              data.rolesAsName = false;
+            }
```

Comment:
The entire conditional logic on lines 772 & 778–780 is optional.

If there are no `roles` then `data.rolesAsName` would remain as `[]` which equates to falsy in the Handlebars.